### ISSUE TITLE
Kernel restart after openning df in Data Wrangler

### ISSUE THREAD

**Issue by zambranolp** [microsoft/vscode-data-wrangler#200]
## Environment data

-   VS Code version: 1.90
-   Data Wrangler Extension version (available under the Extensions sidebar): v1.2.0
-   Jupyter Extension version (available under the Extensions sidebar): 2024.5.0
-   Python Extension version (available under the Extensions sidebar): 2024.8.0
-   OS (Windows | Mac | Linux distro) and version: Win10 Enterprise
-   Pandas version: 1.5.3
-   Python and/or Anaconda version: 3.11.1
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): N/A

## Expected behaviour

Opening a df or .parquet data on a new VSC windows using Data Wrangler
## Actual behaviour

New VSC window open but nothing shows. All jupyter variables of my main notebook are cleared. 

![image](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/7afccefd-6983-4894-a355-4a0dfad3116b)

## Steps to reproduce:

1. Run the necessary jupyter cells to create a df. 
2. Click on "open df in Data Wrangler" option
3. Data Wrangler window opens and seems to load data but a pop-up window showing "Canceled" appears at the right bottom corner and sometimes a "Connection to runtime" shows endlessly. 

![Animation](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/3a3c8c4a-9eaf-47b2-a3a1-a6c52a97cc51)

Thanks

**Comment by pwang347** [comment]
Hi @zambranolp, thanks for opening this issue!

I had some questions to better understand the issue.
1) Is this happening all the time for you, frequently, or just sometimes?
2) Does it matter what sort of data you are opening? (e.g. can you repro it with a simple example like the following code?)

```python
import pandas as pd
df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 4]})
df
```
3) Could you please try making sure this checkbox is unchecked and trying again?
<img width="952" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/67a5ac05-3e0a-4dd5-9bb2-2cb645ae3fc6">

You can find this setting by opening the command palette with `control+shift+P` and then running the following command
<img width="754" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/34b358ee-b20a-4ffe-8ff9-acf14d410ba7">

**Comment by zambranolp** [comment]
Hi @pwang347, thanks for your interest on my post. 

I just left work and will be on holidays until last week of June but in the meantime I can answer some of your questions:

1. it happens all times 
2. It happens with any sort of DataFrame that I created from my notebook and any dataset that I tried to open from the VSC directory (.csv and .parquet). The data I show on the example is actually random data as I couldn’t show the real df (strict confidentiality) but the behaviour is no different. 
3. I clearly remember have all boxes checked, including experimental but I won’t be able to confirm, uncheck it and test it until I’m back from holidays, sorry for that. 

I’ll update as soon as I can. Thanks !

**Comment by pwang347** [comment]
That would be very appreciated @zambranolp! And no worries, we can chat more when you are back.

--

Based on the error message, it seems possible that the kernel is getting interrupted somehow when DW is trying to run code and might be an external issue related to the kernel environment.

If you could please also help try the following items when you are back:
* Disable every VS Code extension except for Jupyter, Python and Data Wrangler
* Please set the logging level to `debug` for Jupyter settings
<img width="479" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/2b000270-81cc-4aab-873a-abc0da1306f5">

* Once you run into the issue again, please share the logs in `Outputs > Jupyter` from the bottom panel (e.g. pasted into a .txt file)
<img width="1195" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/ca479c57-8daa-4e52-b13f-2307e2243f22">


Thanks again, and hope you have a great vacation!

**Comment by zambranolp** [comment]
Hi @pwang347, 

I'm finally back. I did a bit of debug myself and I might have found the issue: the workspace folder name. Here's my experiment: 

I have two folders: 

1. C:\Users\ez\Documents\Debug Level 0\Level 1\Level 2 long long long name
2. C:\Users\ez\Documents\Debug Level 0\Level 1\Level 2 long long long name\Level 3 long long long name

Each of these folders have two notebooks with the exact same content `import pandas as pd
df = pd.DataFrame(data={'col1': [1, 2], 'col2': [3, 4]})
df`

1. Short.ipynb
2. very_long_long_long_name.ipynb

It turns out that in "Level 2..." folder, only "Short.ipynb" shows the df in Data Wrangler as it should be whilst "very_long_long_long_name.ipynb" reproduce the error shown before. Here's a .gif running both files for a better understanding : 

![Debug_2](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/69f35873-2fcd-4061-badd-b267db27b2a4)

The Jupyter debug output of this operation is below: 

```
Visual Studio Code (1.90.2, undefined, desktop)
Jupyter Extension Version: 2024.5.0.
Python Extension Version: 2024.8.1.
Pylance Extension not installed.
Platform: win32 (x64).
Workspace folder ~\Documents\Debug Level 0\Level 1\Level 2 long long long name, Home = c:\Users\s616968
11:17:35.410 [debug] Start refreshing Kernel Picker (1719307055410)
11:17:35.591 [debug] Get Custom Env Variables, Class name = Bm, completed in 181ms, has a truthy return value, Arg 1: undefined, Arg 2: "RunPythonCode"
11:17:35.592 [debug] Jupyter Paths kernels: 
11:17:35.593 [debug] Kernel Spec Root Paths, ~\AppData\Roaming\jupyter\kernels, c:\ProgramData\jupyter\kernels
11:17:35.822 [debug] KernelProvider switched kernel to id = .jvsc74a57bd0d71239005c1cdaa6e1d1c90d837461fc5223d28b4ccd992107b9d31ce6789923.c:\SafApp\Python\Python3.11-64\python.exe.c:\SafApp\Python\Python3.11-64\python.exe.-m#ipykernel_launcher
11:17:35.822 [debug] start the kernel, options.disableUI=true for ~\Documents\Debug Level 0\Level 1\Level 2 long long long name\Short.ipynb
11:17:35.823 [debug] Controller selection change completed
11:17:35.826 [debug] KernelProvider switched kernel to id = .jvsc74a57bd0d71239005c1cdaa6e1d1c90d837461fc5223d28b4ccd992107b9d31ce6789923.c:\SafApp\Python\Python3.11-64\python.exe.c:\SafApp\Python\Python3.11-64\python.exe.-m#ipykernel_launcher
11:17:35.826 [debug] start the kernel, options.disableUI=true for ~\Documents\Debug Level 0\Level 1\Level 2 long long long name\very_long_long_long_name.ipynb
11:17:35.827 [debug] Controller selection change completed
11:17:36.105 [debug] Get Custom Env Variables, Class name = Bm, completed in 0ms, has a truthy return value, Arg 1: "/C:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name", Arg 2: "RunPythonCode"
11:17:36.107 [info] Starting Kernel (Python Path: c:\SafApp\Python\Python3.11-64\python.exe, Unknown, 3.11.1) for '~\Documents\Debug Level 0\Level 1\Level 2 long long long name\Short.ipynb' (disableUI=true)
11:17:36.109 [info] Starting Kernel (Python Path: c:\SafApp\Python\Python3.11-64\python.exe, Unknown, 3.11.1) for '~\Documents\Debug Level 0\Level 1\Level 2 long long long name\very_long_long_long_name.ipynb' (disableUI=true)
11:17:36.173 [debug] Get Custom Env Variables, Class name = Bm, completed in 1ms, has a truthy return value, Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/Short.ipynb", Arg 2: "RunPythonCode"
11:17:36.224 [debug] Get Custom Env Variables, Class name = Bm, completed in 1ms, has a truthy return value, Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/very_long_long_long_name.ipynb", Arg 2: "RunPythonCode"
11:17:36.396 [debug] End refreshing Kernel Picker (1719307055410)
11:17:36.440 [warn] Failed to get activated env vars for c:\SafApp\Python\Python3.11-64\python.exe in 335ms
11:17:36.442 [debug] Create ProcessService, Class name = Vm (started execution), Arg 1: undefined
11:17:36.447 [debug] Get Custom Env Variables, Class name = Bm, completed in 4ms, has a truthy return value, Arg 1: undefined, Arg 2: "RunNonPythonCode", Arg 3: undefined
11:17:36.448 [debug] Create ProcessService, Class name = Vm, completed in 6ms, has a truthy return value, Arg 1: undefined
11:17:36.540 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -c "import site;print("USER_BASE_VALUE");print(site.USER_SITE);print("USER_BASE_VALUE");"
11:17:36.561 [warn] Failed to get activated env vars for c:\SafApp\Python\Python3.11-64\python.exe in 338ms
11:17:36.565 [warn] Failed to get activated env vars for c:\SafApp\Python\Python3.11-64\python.exe in 393ms
11:17:36.566 [debug] Launching kernel .jvsc74a57bd0d71239005c1cdaa6e1d1c90d837461fc5223d28b4ccd992107b9d31ce6789923.c:\SafApp\Python\Python3.11-64\python.exe.c:\SafApp\Python\Python3.11-64\python.exe.-m#ipykernel_launcher for ~\Documents\Debug Level 0\Level 1\Level 2 long long long name\Short.ipynb in ~\Documents\Debug Level 0\Level 1\Level 2 long long long name with ports 9001, 9000, 9004, 9002, 9003
11:17:36.569 [debug] Launching kernel .jvsc74a57bd0d71239005c1cdaa6e1d1c90d837461fc5223d28b4ccd992107b9d31ce6789923.c:\SafApp\Python\Python3.11-64\python.exe.c:\SafApp\Python\Python3.11-64\python.exe.-m#ipykernel_launcher for ~\Documents\Debug Level 0\Level 1\Level 2 long long long name\very_long_long_long_name.ipynb in ~\Documents\Debug Level 0\Level 1\Level 2 long long long name with ports 9009, 9008, 9012, 9010, 9011
11:17:36.876 [debug] Getting activated env variables, Class name = Pp, completed in 771ms, has a truthy return value, Arg 1: "/C:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name", Arg 2: "C:\\SAFAPP\\PYTHON\\PYTHON3.11-64\\PYTHON.EXE", Arg 3: undefined
11:17:36.876 [debug] Getting activated env variables, Class name = Pp, completed in 653ms, has a truthy return value, Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/very_long_long_long_name.ipynb", Arg 2: "C:\\SAFAPP\\PYTHON\\PYTHON3.11-64\\PYTHON.EXE", Arg 3: undefined
11:17:36.876 [debug] Getting activated env variables, Class name = Pp, completed in 704ms, has a truthy return value, Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/Short.ipynb", Arg 2: "C:\\SAFAPP\\PYTHON\\PYTHON3.11-64\\PYTHON.EXE", Arg 3: undefined
11:17:36.949 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -m pip list
11:17:37.017 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
11:17:37.110 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -c "import ipykernel; print(ipykernel.__version__); print("5dc3a68c-e34e-4080-9c3e-2a532b2ccb4d"); print(ipykernel.__file__)"
11:17:37.215 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe c:\Users\~\.vscode\extensions\ms-toolsai.jupyter-2024.5.0-win32-x64\pythonFiles\vscode_datascience_helpers\kernel_interrupt_daemon.py --ppid 35248
    > cwd: ~\.vscode\extensions\ms-toolsai.jupyter-2024.5.0-win32-x64\pythonFiles\vscode_datascience_helpers
11:17:37.996 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -m ipykernel_launcher --f=c:\Users\~\AppData\Roaming\jupyter\runtime\kernel-v2-35248GrV2EimM7GX6.json
    > cwd: ~\Documents\Debug Level 0\Level 1\Level 2 long long long name
11:17:37.996 [debug] Kernel process 32816.
11:17:38.129 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe -m ipykernel_launcher --f=c:\Users\~\AppData\Roaming\jupyter\runtime\kernel-v2-35248bO17jWTgzQgd.json
    > cwd: ~\Documents\Debug Level 0\Level 1\Level 2 long long long name
11:17:38.130 [debug] Kernel process 37788.
11:17:42.487 [debug] Waiting for Raw Session to be ready in postStartRawSession
11:17:42.488 [debug] Waiting for Raw session to be ready, status: connected
11:17:42.490 [debug] Waited for Raw session to be ready & got status: connected
11:17:42.490 [debug] Successfully waited for Raw Session to be ready in postStartRawSession
11:17:42.490 [debug] Kernel status is 'unknown' before requesting kernel info and after ready
11:17:42.490 [debug] Sending request for kernelInfo
11:17:42.515 [debug] Kernel output 32816: To connect another client to this kernel, use:
--existing c:\Users\~\AppData\Roaming\jupyter\runtime\kernel-v2-35248GrV2EimM7GX6.json
11:17:42.658 [debug] Waiting for Raw Session to be ready in postStartRawSession
11:17:42.659 [debug] Waiting for Raw session to be ready, status: connected
11:17:42.660 [debug] Waited for Raw session to be ready & got status: connected
11:17:42.660 [debug] Successfully waited for Raw Session to be ready in postStartRawSession
11:17:42.661 [debug] Kernel status is 'unknown' before requesting kernel info and after ready
11:17:42.661 [debug] Sending request for kernelInfo
11:17:42.726 [debug] Successfully completed postStartRawSession after 1 attempt(s) in 236ms
11:17:42.857 [debug] Kernel output 37788: To connect another client to this kernel, use:
--existing c:\Users\~\AppData\Roaming\jupyter\runtime\kernel-v2-35248bO17jWTgzQgd.json
11:17:42.890 [debug] Requesting Kernel info
11:17:42.895 [info] Kernel successfully started
11:17:42.986 [debug] Successfully completed postStartRawSession after 1 attempt(s) in 325ms
11:17:43.084 [debug] Requesting Kernel info
11:17:43.087 [info] Kernel successfully started
11:17:46.368 [debug] Start refreshing Interpreter Kernel Picker
11:17:46.368 [debug] Start refreshing Kernel Picker (1719307066368)
11:17:46.439 [debug] getDataDirsImpl, Class name = jr (started execution), Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/very_long_long_long_name.ipynb", Arg 2: "C:\\SAFAPP\\PYTHON\\PYTHON3.11-64\\PYTHON.EXE"
11:17:46.442 [debug] Get Custom Env Variables, Class name = Bm, completed in 3ms, has a truthy return value, Arg 1: undefined, Arg 2: "RunPythonCode"
11:17:46.442 [debug] Jupyter Paths : 
11:17:46.517 [info] Process Execution: c:\SafApp\Python\Python3.11-64\python.exe c:\Users\~\.vscode\extensions\ms-toolsai.jupyter-2024.5.0-win32-x64\pythonFiles\printJupyterDataDir.py
11:17:46.864 [debug] getDataDirsImpl, Class name = jr, completed in 425ms, has a truthy return value, Arg 1: "/c:/Users/~/Documents/Debug Level 0/Level 1/Level 2 long long long name/very_long_long_long_name.ipynb", Arg 2: "C:\\SAFAPP\\PYTHON\\PYTHON3.11-64\\PYTHON.EXE", Return Value: <Uri:/C:/Users/~/AppData/Roaming/Python/share/jupyter>, <Uri:/C:/Users/~/AppData/Roaming/jupyter>, <Uri:/C:/SafApp/Python/Python3.11-64/share/jupyter>, <Uri:/C:/ProgramData/jupyter>
11:18:01.331 [debug] Controller startUsingPythonInterpreter:'.jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher' for view = 'jupyter-notebook' is no longer a valid
11:18:01.331 [debug] Controller startUsingPythonInterpreter:'.jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher (Interactive)' for view = 'interactive' is no longer a valid
11:18:01.331 [warn] Disposing old controller startUsingPythonInterpreter:'.jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher' for view = 'jupyter-notebook'
11:18:01.331 [debug] Disposing controller .jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher associated with connection .jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher 
11:18:01.332 [warn] Disposing old controller startUsingPythonInterpreter:'.jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher (Interactive)' for view = 'interactive'
11:18:01.332 [debug] Disposing controller .jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher (Interactive) associated with connection .jvsc74a57bd06a93d538fe142fb12988c4e4306561f63b3ede3729d6d3378f6ffd64c4d70ffc.c:\Users\~\Anaconda3\python.exe.c:\Users\~\Anaconda3\python.exe.-m#ipykernel_launcher 
11:18:01.343 [debug] Refreshed Environments
11:18:02.336 [debug] Interpreter removed C:\USERS\~\ANACONDA3\PYTHON.EXE
11:18:02.351 [debug] End refreshing Kernel Picker (1719307066368)
```

Funny thing: for folder "Level 3...", **both** "Short.ipynb" and "very_long_long_long_name.ipynb" crashes when trying to launch Data Wrangler.

**Comment by pwang347** [comment]
Hi @zambranolp, thanks for the reply! I just got time to take a look at this again. What is very curious is that the crash occurs only for launching Data Wrangler, and not in the notebook. I was able to reproduce a similar issue but with a path length of ~257 characters which is [expected](https://discuss.python.org/t/error-if-full-path-to-file-more-than-256-symbols/33794/4). Yours however seems to repro much lower than that (<100).

Additionally, this is the error I get, which is different:
<img width="960" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/49e306ab-c009-4434-b53e-81f6eb1ea683">

I see that "Canceled" is also appearing for me, but it's only in the logs (and your logs do not include it). Based on the above, it is highly likely that the crash is happening on some code we are running.

Could you please try the following?
1. Enable the `debug` option in Data Wrangler settings
<img width="484" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/9c447b01-7593-48f4-b8be-b96c852c89cf">

2. Reproduce the issue again
3. Check the following Output channel for the code execute
<img width="563" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/5b8727ef-c3a5-4550-bb82-64a216d3718b">

4. Attach the results in the channel also into a txt file.

Thank you!

**Comment by zambranolp** [comment]
Hello again @pwang347! 

I've got a question on that: the Data Wrangler Kernel Output option only becomes visible after effectively launching DW (I did it for the "short.ipynb" file). This means that I can reproduce the issue with the "long long... .ipynb" but once it crashes, the output option is no longer available on the menu. Or did I missunderstood the steps ? Sorry in that case :) 

Regarding the optional step, I did run the blocks from top to bottom on my last .gif reproducing the error but not sure if that's what you meant. 

Thanks!

**Comment by pwang347** [comment]
Thanks for checking @zambranolp!

Interesting, I don't think the kernel output option is supposed to disappear even on crashes (so maybe it never appeared). It seems unlikely then that the issue is the Python code being executed.

Could you please also check the console output for any errors by following these steps:
<img width="398" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/ceb9c72c-33ee-4f78-865a-32fe8ac9e47c">

And look for any error messages here:
<img width="383" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/cd609494-cfac-4a2c-9bb4-7a201ab22b74">

Thanks!

**Comment by zambranolp** [comment]
Hi @pwang347

I don't really know how to share the console output for you to have the best visualisation of the messages, so I'll just share both screenshots and text file (although the text file is a bit chaotic). The outputs are from the moment I declare the variables and the errors start at the moment I launch DW. 

Thanks for your help.

Attachements -------------------
![console_1](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/2e630737-1bfb-4f78-b736-69a488aed4f9)
![console_2](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/a2b1b854-bca5-4d91-833a-0edd98d085fb)
![console_3](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/a69a3ec8-03a9-4cbc-af91-5b39291f6fdb)
![console_4](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/7db0d598-f363-4977-8d8c-a296d798ec7b)
The next one was missing just the last bit:
![console_5](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/074f5bc0-0577-4078-bc30-d31194bd9a80)
[VSC_dev_tool_console.txt](https://github.com/user-attachments/files/16048969/VSC_dev_tool_console.txt)

**Comment by pwang347** [comment]
Hi @zambranolp, thank you very much for sharing the logs!

This part looks very suspect; the VS Code extension host crashed and it looks like it may have caused the Jupyter kernel connection to close:
<img width="1286" alt="image" src="https://github.com/microsoft/vscode-data-wrangler/assets/8560030/fb691812-1433-4034-8be5-3c32219b6dea">

In general, the VS Code extension host shouldn't crash from regular extension code, but for things like out of memory or other low-level issues. Could you please try to get more information about the crash following the steps here? https://github.com/microsoft/vscode/issues/176562#issuecomment-1470150459

Thanks again!

**Comment by zambranolp** [comment]
Hello @pwang347, 

Here's a .zip containing the reports generated while reproducing the issue: 
[20240702T092121.zip](https://github.com/user-attachments/files/16063862/20240702T092121.zip)

Thanks a lot for your help.

**Comment by pwang347** [comment]
Hi @zambranolp! Great news, we were able to identify the issue with the information you provided.

Turns out this is caused by a combination of enabling the Data Wrangler debug setting, using a Windows machine and having a relatively long path (~100 characters).

The crash itself is caused by this upstream VS Code issue: https://github.com/microsoft/vscode/issues/219785

We've released a fix in the latest pre-release version (1.5.2) which should address the issue on our side. Alternatively, if you want to remain on your current version you should be able to work around this by disabling the debug setting. Please give either of these a try and let me know if the issue is resolved for you as well.

Thanks again for the patience and for helping with the investigation!

**Comment by zambranolp** [comment]
Hi @pwang347 

Pre-release version worked like a charm. Thank you so much for your work and effort. DW is definitely a life changer for my workflow and coding style, so I was a bit sad not being able to use it. Hats off for you and the team behind this. 

![image](https://github.com/microsoft/vscode-data-wrangler/assets/45477198/408ab872-eac3-4992-9b9c-f6f1546fd64d)
